src/dashboard-refactor/styled-components.tsx

```diff
@@ -42,3 +45,35 @@ export const Icon = styled.div<IconProps>`
             animation: ${rotate(-90)} 1s ease-in 1;
         `}
 `
+
+interface LoadingContainerProps {
+    top?: string
+    bottom?: string
+    left?: string
+    right?: string
+    horizontal?: string
+    vertical?: string
+    backgroundColor: string
+}
+
+export const renderLoadingContainer = (props: LoadingContainerProps) => {
```

Comment:
Is there any reason for this not to follow standard naming conventions? Something like `function LoadingContainer`?

Reply:
Do you mean that I should create this just as a React Functional Component?

Reply:
Yup, because it seems to be one.